cachi2/core/package_managers/rpm/main.py

```diff
@@ -286,9 +286,9 @@ def _resolve_rpm_project(source_dir: RootedPath, output_dir: RootedPath) -> list
 
 def _download(lockfile: RedhatRpmsLock, output_dir: Path) -> dict[Path, Any]:
     """
-    Download packages mentioned in the lockfile.
```

Comment:
**Commit message:**

Please don't include the JIRA IDs in commit messages, they don't help anyone without access, so if those provide any further context it won't be available publicly.